app/src/configurable_auth.h

```diff
@@ -4,6 +4,7 @@
 #pragma once
 
 #include "kv_types.h"
+#include "tracing.h"
```

Comment:
Given that logging in authn policy classes runs before the actual endpoint starts there's not really a reason to use our custom logging macros here (as it wouldn't add request id or client request id anyway). By using `CCF_..` we would avoid this extra include and also the `inline`s everywhere.

Reply:
fixed